chart/templates/operator.yaml

```diff
@@ -288,40 +287,54 @@ spec:
       labels:
         app: status-controller
     spec:
+      securityContext:
+        runAsNonRoot: true
+      serviceAccountName: status-sa
+      initContainers:
+      - name: create-crd
+        image: quay.io/kubestellar/kubectl:1.29.3
+        command: ['sh', '-c']
+        args:
+        - |
+          echo -n "Waiting for managedclusteraddons CRD"
+          while [ "$(kubectl get crd clustermanagementaddons.addon.open-cluster-management.io 2> /dev/null)" == "" ] ; do
+            echo -n "."
+            sleep 1
```

Comment:
I would be more relaxed about this, sleep for 5 or 10 seconds.

Reply:
changed to 5, I'm impatient